### PR TITLE
docs: Fix minor issues in getting started guide

### PR DESCRIPTION
**Pull request by twpayne**
Please ensure your pull request adheres to the following guidelines:

- [x] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [x] All code is covered by unit and/or runtime tests where feasible.
- [x] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [x] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [x] Provide a title or release-note blurb suitable for the release notes.
- [x] Thanks for contributing!

This PR corrects a few minor issues found while working through the getting started guide. See the individual commits for descriptions.

### CODE REVIEW

**Review by kkourt — APPROVED** [review]

File: Documentation/gettingstarted/k8s-install-connectivity-test.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ indicates success or failure of the test:
 
 .. code:: shell-session
 
-   $ kubectl get pods -n cilium-test
+   $ kubectl get pods
```
Comment (rolinh) [inline Documentation/gettingstarted/k8s-install-connectivity-test.rst]: The [troubleshooting guide](https://docs.cilium.io/en/latest/operations/troubleshooting/#cilium-connectivity-tests) recommends deploying the connectivity test into a `cilium-test` namespace. I think we should aim for consistency. What about adding a step above to `kubectl create ns cilium-test` instead?

Comment (twpayne) [reply Documentation/gettingstarted/k8s-install-connectivity-test.rst]: SGTM, I'll update the this PR to include that.

**Review by pchaigno — APPROVED** [review]
LGTM once Robin's comment is addressed.

File: Documentation/gettingstarted/k8s-install-connectivity-test.rst (hunk 1 above)
Comment (twpayne) [reply Documentation/gettingstarted/k8s-install-connectivity-test.rst]: PR updated.

**Review by aanm — APPROVED** [review]

**Review by rolinh — APPROVED** [review]

**Pull request merged**